hermes-mock/src/main/java/pl/allegro/tech/hermes/mock/HermesMockDefine.java

```diff
@@ -0,0 +1,31 @@
+package pl.allegro.tech.hermes.mock;
+
+import org.apache.http.HttpStatus;
+
+class HermesMockDefine {
```

Comment:
Maybe we can return in all methods `this`. Then we could use it as a builder like this
```
hermesMock.define()
    .jsonTopic("asd")
    .avroTopic("bcd")
```
What do you think?

Reply:
Good point!